locales/az/az.json

```diff
@@ -589,7 +589,7 @@
     "Thailand": "Tayland",
     "Thanks for signing up! Before getting started, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.": "Qeydiyyatdan keçdiyiniz üçün təşəkkür edirik! Siz başlamaq əvvəl, Biz yalnız e-poçt vasitəsilə göndərilir linki tıklayarak e-poçt ünvanınızı təsdiq edə bilər? Bir məktub almadıysanız, biz məmnuniyyətlə Başqa bir şey göndərəcəyik.",
     "Thanks for your continued support. We've attached a copy of your invoice for your records. Please let us know if you have any questions or concerns.": "Thanks for your continued support. We've attached a copy of your invoice for your records. Please let us know if you have any questions or concerns.",
-    "Thanks,": "Thanks,",
+    "Thanks,": "təşəkkürlər,",
```

Comment:
Is it correct that the first character of a word is lowercase?

Reply:
I will correct my mistake and send

Reply:
```suggestion
    "Thanks,": "Təşəkkürlər,",
```